test/screenshot/spec/mdc-textfield/classes/invalid-focused-helper-text-persistent-validation-msg.html

```diff
@@ -41,7 +41,7 @@
 
     <div class="test-cell test-cell--textfield">
       <div class="mdc-text-field mdc-text-field--outlined mdc-text-field--invalid mdc-text-field--focused">
-        <input type="text" id="outlined-text-field" class="mdc-text-field__input test-text-field__input" value=" ">
+        <input type="text" id="outlined-text-field" class="mdc-text-field__input test-text-field__input" value=" " required pattern=".{4,8}">
```

Comment:
Is the space in `value=" "` intentional?

If so, maybe add an HTML comment explaining what it's for.

Reply:
I added a comment to explain. The text field does not draw the SVG path when the label is floating unless the input has a value or has focus. Since I'm forcing the focused state for the purpose of screenshot tests, I added a single space value to force the outline to notch.